app/configurations/config.herrenberg.js

```diff
@@ -397,8 +396,6 @@ export default configMerger(parentConfig, {
         'lot_type:Parkhaus'
     ],
 
-    realtime: { hbg: realtimeHbg },
-
```

Comment:
Are you sure, that this should be removed?

Reply:
See Line: 238